libnmstate/netapplier.py

```diff
@@ -360,9 +358,21 @@ def _sort_lag_slaves(desired_state, current_state):
 
 
 def _remove_absent_iface_entries(ifaces_desired_state):
-    return {ifname: ifstate
-            for ifname, ifstate in six.viewitems(ifaces_desired_state)
-            if ifstate.get('state') != 'absent'}
+    ifaces = {}
+    for ifname, ifstate in six.viewitems(ifaces_desired_state):
+        iface_admin_state = ifstate.get('state')
+        is_absent = iface_admin_state == 'absent'
+        is_virt_down = (
+            iface_admin_state == 'down' and
+            ifstate.get('type') in ('bond',
```

Comment:
IMHO the list of virtual interfaces should be a constant or this check should be its own method to ensure that we do not have multiple lists of virtual types.

Reply:
Makes sense. I will move it to the schema module.